apps/test-app/frontend/src/index.tsx

```diff
@@ -80,10 +80,14 @@ async function initializePresentation() {
   });
   // __PUBLISH_EXTRACT_END__
 
+  /* eslint-disable @typescript-eslint/no-deprecated */
```

Comment:
Doesn't look like we're using this extraction anymore and we probably don't need the active scope here. Let's remove it.